surveys/serializer_field_generators.py

```diff
@@ -4,6 +4,31 @@
 from .generators import BaseFieldGenerator as BaseBaseFieldGenerator
 
 
+class BlankIntegerField(serializers.IntegerField):
```

Comment:
I think we need the same / similar functionality for all field generators based on `IntegerFields`  as the frontend will send an empty string when there is no value.